instantmusic-0.1/bin/instantmusic

```diff
@@ -114,7 +114,7 @@ def query_and_download(search, has_prompts=True, is_quiet=False):
     # We only ask the user which one they want if prompts are on, of course
     if has_prompts and not is_quiet:
         choice = ''
-        while choice.strip() == '':
+        while not(choice.isdigit()) or not(0 <= int(choice) < len(available)):
```

Comment:
No, the first `not` is only for the `choice.isdigit()` and not for the whole `while`:

<pre>while not<b>(</b>choice.isdigit()<b>)</b> or not<b>(</b>0 <= int(choice) < len(available)<b>)</b>:</pre>